meta-avocado/classes/package-image.bbclass

```diff
@@ -0,0 +1,25 @@
+# Class to package a rootfs image into a package
+
+AVOCADO_PKG_IMG_NAME ?= ""
+AVOCADO_PKG_IMG_RECIPE ?= ""
+
+PACKAGE_ARCH = "${MACHINE_ARCH}"
+PACKAGES = "${PN}"
+INSTALL_PREFIX = "${SDKPATHINSTALL}/images"
+
+python () {
+    if not d.getVar('AVOCADO_PKG_IMG_NAME') or d.getVar('AVOCADO_PKG_IMG_NAME') == "":
+        bb.fatal("AVOCADO_PKG_IMG_NAME must be set in the recipe using this class.")
+    if not d.getVar('AVOCADO_PKG_IMG_RECIPE') or d.getVar('AVOCADO_PKG_IMG_RECIPE') == "":
+        bb.fatal("AVOCADO_PKG_IMG_RECIPE must be set in the recipe using this class.")
+    d.setVarFlag('do_compile', 'depends', d.getVar('AVOCADO_PKG_IMG_RECIPE') + ":do_image_complete")
```

Comment:
Very clever. I struggled make this work in my earlier attempts at packaging images.